GitCommands/CommitDataManager.cs

```diff
@@ -200,7 +203,11 @@ private bool TryGetCommitLog(string commitId, string format, [NotNullWhen(return
             };
 
             // Do not cache this command, since notes can be added
-            data = GetModule().GitExecutable.GetOutput(arguments, outputEncoding: GitModule.LosslessEncoding);
+            data = ThreadHelper.JoinableTaskFactory.Run(async () =>
+            {
+                await TaskScheduler.Default;
+                return GetModule().GitExecutable.GetOutput(arguments, outputEncoding: GitModule.LosslessEncoding);
+            });
```

Comment:
Why is this changing? `Run` blocks the current thread exactly the same way that calling `GetOutput` directly blocks the current thread. This form is both more difficult to read and requires two threads to do the work that previously only needed one.

Reply:
reverting this and similar changes. 
I expected the UI thread not to be blocked by these changes, available for other actions.